1_fetch.R

```diff
@@ -462,6 +463,14 @@ p1_targets_list <- list(
     },
     format = 'file', 
     repository = 'local'
+  ),
+  #save as npz for use in Python
```

Comment:
Thanks for adding the npz target! Do any of our other inputs need to be formatted as npz, or just the distance matrix?

Reply:
Just the distance matrix. The other inputs are attributes and SC observations, which are both saved as zarr files.

Reply:
Sounds good, thanks for the reminder about the expected file formats!